### PR TITLE
Use Zenaton Gateway instead of the local agent.

### PR DESCRIPTION
**Pull request by MrYawe**
### Safety check

| Q                         | A
| ------------------------- | ---
| Type                      |enhancement
| Related issues            | #...
| Code covered with tests?  | yes/no
| Changelog updated?        | yes/no
| Documentation updated?    | yes/no

### CODE REVIEW

File: zenaton/client.py [hunk 1]
```diff
@@ -200,52 +168,83 @@ def send_event(self, workflow_name, custom_id, event):
...
+                self.serializer.decode(res['properties']),
                 Workflow
             )
-        else:
-            return None
 
     """
         Stops a workflow
-        :param String workflow_name the class name of the workflow
+        :param .abstracts.workflow.Workflow flow
```
Comment (idabmat) [inline zenaton/client.py]: I don't know how the documentation in Python works. But shouldn't the last value here `flow` match the actual variable name `workflow` ?

File: zenaton/services/graphql_service.py [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
 import requests
 
 
 class GraphQLService:
+
+    # Queries
+    FIND_WORKFLOW = """
+        query ($custom_id: String!, $environment_name: String!, $programming_language: ProgrammingLanguage!, $name: String!) {
+            findWorkflow(custom_id: $custom_id, environment_name: $environment_name, programming_language: $programming_language, name: $name) {
+                id
```
Comment (idabmat) [inline zenaton/services/graphql_service.py:12]: We don't return the `id` in php and node. Not sure it is important though ...

**Review by idabmat — APPROVED** [review]

File: zenaton/services/graphql_service.py (hunk 2 above)
Comment (pylebecq) [reply zenaton/services/graphql_service.py:12]: It's not. We won't return anything to the user in any case.
It's just that GraphQL requires you to select one field. We just picked a different field ^^

File: zenaton/client.py (hunk 1 above)
Comment (MrYawe) [reply zenaton/client.py]: Yes 👍

**Pull request merged**